src/blueapi/service/authentication.py

```diff
@@ -141,8 +144,6 @@ def _do_device_flow(self) -> None:
             self._server_config.device_authorization_endpoint,
             data={
                 "client_id": self._server_config.client_id,
-                "scope": "openid profile offline_access",
```

Comment:
Did we decide this wasn't required for the refresh to work? Or was it still needed?